process_report/process_report.py

```diff
@@ -387,6 +389,11 @@ def upload_to_s3_old_pi_file(old_pi_file):
     invoice_bucket.upload_file(old_pi_file, PI_S3_FILEPATH)
 
 
+def backup_to_s3_old_pi_file(old_pi_file):
+    invoice_bucket = get_invoice_bucket()
+    invoice_bucket.upload_file(old_pi_file, f"PIs/Archive/PI {get_iso8601_time()}.csv")
```

Comment:
`backup_to_s3_old_pi_file` and `upload_to_s3_old_pi_file` can be one method and you can just pass the s3 filepath as an argument, but you can do that as part of refactoring.

Reply:
I'll do that as part of refactoring later.